pkg/component/manager_test.go

```diff
@@ -52,29 +52,32 @@ func Test_ResolvePath(t *testing.T) {
 		test.StageFile(t, fs, "deployment.yaml", "/app/components/nested/deployment.yaml")
 
 		cases := []struct {
-			name   string
-			cName  string
-			module string
-			isErr  bool
+			name              string
+			expectedComponent string
+			expectedModule    string
+			isErr             bool
 		}{
 			{
-				name:   "/",
-				module: "/",
+				name:           "/",
+				expectedModule: "/",
 			},
 			{
-				name:   "deployment",
-				module: "/",
-				cName:  "deployment",
+				name:              "deployment",
+				expectedModule:    "/",
+				expectedComponent: "deployment",
 			},
 			{
-				name:   "deployment",
-				module: "/",
-				cName:  "deployment",
+				name:              "nested.deployment",
+				expectedModule:    "nested",
+				expectedComponent: "deployment",
 			},
 			{
-				name:   "nested/deployment",
-				module: "nested",
-				cName:  "deployment",
+				name:           "nested",
```

Comment:
this test is not nested and the case is also covered in "deployment" we should remove this test